ControlRoomApplication/ControlRoomApplication/GUI/RTControlForm.designer.cs

```diff
@@ -968,9 +980,11 @@ private void InitializeComponent()
             this.Controls.Add(this.overRideGroupbox);
             this.Controls.Add(this.errorLabel);
             this.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedSingle;
-            this.MinimumSize = new System.Drawing.Size(600, 478);
+            this.Margin = new System.Windows.Forms.Padding(4, 4, 4, 4);
+            this.MinimumSize = new System.Drawing.Size(794, 577);
             this.Name = "FreeControlForm";
             this.Text = "Control Form";
+            this.Load += new System.EventHandler(this.FreeControlForm_Load);
```

Comment:
Same as in [this](https://github.com/YCPRadioTelescope/YCP-RT-ControlRoom/pull/440) PR.

- [ ] Remove call to empty event handler